hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMOpenKeysDeleteRequest.java

```diff
@@ -97,7 +97,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       }
 
       omClientResponse = new OMOpenKeysDeleteResponse(omResponse.build(),
-          deletedOpenKeys, ozoneManager.isRatisEnabled());
+          deletedOpenKeys, ozoneManager.isRatisEnabled(), getBucketLayout());
```

Comment:
Thanks @kaijchen working on this.  Does this patch just add getBucketLayout? I don't see where getBucketLayout is used in this patch. Do we handle FSO logic for this getBucketLayout?

Reply:
The `BucketLayout` here is only used to indicate which table is the open keys from.

Reply:
Please see line 159 and 167 in this file.

Reply:
The old code will only return `DEFAULT`, and the new code allows us to set it to FSO if need.

Reply:
Understand, we through omMetadataManager.getOpenKeyTable(getBucketLayout()) get the right openkey.